gcc/rust/analysis/rust-hir-liveness.cc

```diff
@@ -114,7 +110,10 @@ Liveness::visit (HIR::PathInExpression &expr)
 	  rust_error_at (expr.get_locus (), "reverse lookup failure");
 	  return;
 	}
-      if (scannedSymbols.find (ref) != scannedSymbols.end ())
+      for (auto s : scannedSymbols)
+	{
```

Comment:
is this a loop with an empty body?

Reply:
Yeah, the gcc code indent style really makes me confused about my own code. :(
I have to spend more time adapting.

Reply:
Yeah i know what you mean are you able to take advantage of the clang-format style it helps me.